Content.Shared/Cargo/CargoOrderData.cs

```diff
@@ -1,27 +1,34 @@
 using Robust.Shared.Serialization;
-
+using Content.Shared.Access.Components;
 namespace Content.Shared.Cargo
 {
     [NetSerializable, Serializable]
     public sealed class CargoOrderData
     {
         public int OrderNumber;
+        public string ProductId;
+        public int Amount;
         public string Requester;
         // public String RequesterRank; // TODO Figure out how to get Character ID card data
         // public int RequesterId;
         public string Reason;
-        public string ProductId;
-        public int Amount;
         public bool Approved;
         public string Approver = string.Empty;
+        public string ApproverName = string.Empty;
+        public string ApproverJob = string.Empty;
```

Comment:
You could probably also make the Approver a nullable string and then Approved just checks if it's not null.

Reply:
Are you suggesting something like that?
```
public bool Approved;
{
    get { return Approver is null; }
}
```

Reply:
yeah, or just inline it like `public bool Approved => Approver != null;`